optimum/commands/export/neuronx.py

```diff
@@ -132,6 +132,36 @@ def parse_args_neuronx(parser: "ArgumentParser"):
         action="store_true",
         help=("Whether or not for the traced model to return the hidden states of all layers."),
     )
+    optional_group.add_argument(
+        "--lora_model_ids",
+        default=[],
```

Comment:
As good practice I would rather set the default to `None` and instantiate a list if the argument is `None`.